src/_locales/de/messages.yml

```diff
@@ -148,7 +148,7 @@ labelAuthor:
   message: 'Autor: '
 anchorAuthor:
   description: Author shown on about tab.
-  message: '<a href=mailto:i@gerald.top>Gerald</a>'
+  message: ''
```

Comment:
Why is this removed?

Reply:
Your adress and name needs no translation, and by delegating this field back to the English locale, a change there gets displayed in the German version without having to edit its locale.

Reply:
OK, I am just curious since it makes no change to the result. 😄